apis/hive/v1/hiveconfig_types.go

```diff
@@ -88,9 +88,56 @@ type HiveConfigSpec struct {
 	// +optional
 	ControllersConfig *ControllersConfig `json:"controllersConfig,omitempty"`
 
+	AWSPrivateLinkController *AWSPrivateLinkControllerConfig `json:"awsPrivateLinkController,omitempty"`
+
 	FeatureGates *FeatureGateSelection `json:"featureGates,omitempty"`
 }
 
+// AWSPrivateLinkControllerConfig defines the configuration for the controller.
```

Comment:
if these are requires, the more appropriate place is controller's package docs. but since there are no other packages that use that in the repo.. i'll defer to the use/configure doc that is already required.